packages/ui/src/components/DatePicker/PreviousDateRangePicker.tsx

```diff
@@ -60,6 +66,74 @@ function toDateTimeString(date: Date, showYear: boolean) {
 	return date.toLocaleDateString('en-us', options)
 }
 
+/**
+ * Takes in a time string in the format HH:mm aa or HH:mmaa and returns	an object with the hour, minute, time of day, and 24 hour time
+ * @param timeString
+ * @returns {	hour: number, minute: number, timeOfDay: string, hour24: number}
+ */
+export const getTimeInfo = (timeString: string) => {
+	const time = timeString.slice(0, -2).trim()
+	const period = timeString.slice(-2).trim().toUpperCase()
+	const [hour, minute] = time.split(':')
+	const isAM = period === 'AM'
+	const isPM = period === 'PM'
+	const hourInt = parseInt(hour, 10)
+	const hour24 =
+		isPM && hourInt !== 12
+			? hourInt + 12
+			: isAM && hourInt === 12
+			? 0
+			: hourInt
+	const timeOfDay = isAM ? 'AM' : isPM ? 'PM' : ''
+
+	return { hour: hourInt, minute: parseInt(minute, 10), timeOfDay, hour24 }
+}
```

Comment:
could we use moment.js for this: https://momentjs.com/docs/#/parsing/
this already exists in moment, ie something like:
```js
let m = moment('12:34 pm', 'hh:mm a');
let data = { hour: m.format('HH'), minute: m.minute(), timeOfDay: m.hour() >= 12, hour24: m.hour() };
console.log({ data });
```